### ISSUE TITLE
Table expanded: last column padding thrown off by expanded row 

### ISSUE THREAD

**Issue by lyubomir-popov** [canonical-web-and-design/vanilla-framework#2503]
## Pattern to amend

Table padding meant to appear between columns also appears at the end of the table.
This happens, because the selector doesn't take into account the presence of the invisible last child expanding row:
![image](https://user-images.githubusercontent.com/2741678/63344902-6d94ad80-c349-11e9-922a-48f4d4fda680.png)
IN the case of expanded tables, we should remove this padding on the last _visible_ cell, not the last child.

**Comment by Caleb-Ellis** [comment]
I have this override in `maas-ui`:
``` scss
.p-table-expanding th,
.p-table-expanding td {
  &:nth-last-child(2) {
    padding-right: $sph-inner--small;
  }
}
```

**Comment by lyubomir-popov** [comment]
This is fixed in 2.4, closing
